test/e2e/antreapolicy_test.go

```diff
@@ -1847,7 +1859,24 @@ func applyPolicies(t *testing.T, step *TestStep) {
 	}
 }
 
-func applyClusterGroups(t *testing.T, step *TestStep) {
+func cleanupTestStepPolicies(t *testing.T, step *TestStep) {
+	for _, np := range step.Policies {
+		if acnp, ok := np.(*secv1alpha1.ClusterNetworkPolicy); ok {
+			failOnError(k8sUtils.DeleteACNP(acnp.Name), t)
+		} else if anp, ok := np.(*secv1alpha1.NetworkPolicy); ok {
+			failOnError(k8sUtils.DeleteANP(anp.Namespace, anp.Name), t)
+		} else {
+			k8sNP, _ := np.(*v1net.NetworkPolicy)
+			failOnError(k8sUtils.DeleteNetworkPolicy(k8sNP.Namespace, k8sNP.Name), t)
+		}
```

Comment:
nit: this could be written as a switch statement

Reply:
Sure. Refactored.